### PR TITLE
RET-2612: Email on case rejection document upload

### PR DESCRIPTION
**Pull request by JackReevies**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-2612

### Change description ###

added aboutToSubmit callback for uploadDocument

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: definitions/json/CaseField.json [hunk 1]
```diff
@@ -3453,6 +3453,13 @@
     "FieldType": "hubLinksStatuses",
     "SecurityClassification": "Public"
   },
+  {
+    "CaseTypeID": "ET_Scotland",
+    "ID": "caseRejectedEmailSent",
+    "Label": " ",
+    "FieldType": "TextArea",
```
Comment (hsjhita) [inline definitions/json/CaseField.json:3460]: TextArea? Think in the code you put it as Yes, would a YesNo make more sense

Comment (hsjhita) [reply definitions/json/CaseField.json:3460]: And authorisations

Comment (hsjhita) [reply definitions/json/CaseField.json:3460]: TextArea does work as well but surely a yes no makes sense?

Comment (JackReevies) [reply definitions/json/CaseField.json:3460]: We don't display this in the frontend, its just so that we don't get a field unknown error on case submit

**Review by hsjhita — APPROVED** [review]

**Review by Brogie — APPROVED** [review]

**Pull request merged**